pkg/systemd/overview.jsx

```diff
@@ -271,7 +271,7 @@ class OverviewPage extends React.Component {
                             { headerActions }
                         </div>
                     </PageSection>
-                    <PageSection variant={PageSectionVariants.default}>
+                    <PageSection variant={PageSectionVariants.default} padding={{ default: 'noPadding', sm: 'padding' }}>
```

Comment:
This looks the wrong way around, but does work perfectly. Nice!

Reply:
I guess it's "mobile first" saying no padding by default, but add padding for size `sm` and up?